src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
Rewrite `main` with `for`:
`
for(int i = 0; i < 3; i++) {
System.out.println(new Lottery().getRandomBall().toString());
}
`